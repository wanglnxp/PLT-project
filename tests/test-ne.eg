int main(){
	int x = 7;
	float y = 3.1;
	print(-x);
	print(-y);
	x = -7;
	y = -3.1;
	print(x);
	print(-x);
	print(y);
	print(-y);
	return 0;
}

