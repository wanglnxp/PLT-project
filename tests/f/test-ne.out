-7
-3.100000
-7
7
-3.100000
3.100000
